src/07-even-more-loops.js

```diff
@@ -143,7 +229,10 @@ function findEqual(array, selected) {}
  *  //> [ 1, 11, 2, 3, 4, 9 ]
  */
 
-function removeDuplicates(array) {}
+function removeDuplicates(array) {
```

Comment:
Lets do a refactor of this without using `new Set` or a spread operator.  It's very important we know what each line of a code is doing at all times.